guides.md

```diff
@@ -28,6 +29,145 @@ our values.
+9. From your own @hypha.com email, send a welcome email, like this template:
+    >
+    > Hi XXXX --
+    >
+    > Yay! Your Hypha email's set up and forwarding to your personal inbox. People can start emailing you at this address immediately, without any extra effort on your part.
+    >
+    > Your new email (and login): XXX@hypha.coop  
+    > Your password: Sent via Signal to XXX-XXX-XXXX
+    >
+    > Having said that, when you'd like to start _sending_ email as XXX@hypha.coop, you'll need to take some more steps.
+    >
+    > The next steps will depend on how you like to handle email:
+    > - [If you'd like to use it from your existing Gmail account...]()
+    > - [If you'd like to use it from our hosted webmail interface...]()
+    > - [If you'd like to use it from another email client...]()
+    >
+    > Once you're set up, let us know by sending a test email to infra@hypha.coop, ideally with a short emoji-only story. (Keypad mashes a-ok!) We'll send a friendly poke in a week if we haven't heard back.
+    >
+    > Questions? Feedback? Reach out via infra@hypha.coop or [`#hyphacoop-infra-ops:tomesh.net`](https://chat.tomesh.net/#/room/#hyphacoop-infra-ops:tomesh.net) for chat.
+    >
+    > In Solidarity,  
+    > xxxxx
+10. Send a low-entropy Signal message with the previously noted password, for example: `🍄 password: xxxxxxxx`
+
+### Using your new inbox (Users)
+
+Accessing your inbox can be done via the hosted webmail interface or by using the same desktop email client you're already using.
+
+What you'll need:
+- Your new email address
+- Your password (sent to you by admin on initial setup)
+
+#### Using GMail
+
+1. Log into GMail
+2. Go to GMail settings, by navigating:  
+    _:gear: (top-right) > Settings > Accounts and Import_
+3. Under "Send mail as", click "Add another email address".
+    <details>
+      <summary>View Screenshot</summary>
+      <img alt="Screenshot of GMail settings page" src="https://i.imgur.com/oQMdIS9.png" />
+    </details>
+5. In the new popup, enter:
+    - _Name_: as will appear to email recipients
+    - _Email address_: new email address e.g., `handle@hypha.coop`
+    - _Treat as alias_: checked :ballot_box_with_check:
+    <details>
+      <summary>View Screenshot</summary>
+      <img alt="Screenshot of new alias popup" src="https://i.imgur.com/I2RGv80.png" />
+    </details>
+4. Click "Next", then:
+    - _SMTP Server_: `mailninja.aseriesoftubez.com` (auto-filled)
+    - _Port_: `587` (auto-filled)
+    - _Username_: replace with new email address, e.g., `handle@hypha.coop`
+    - _Password_: your password
+    - TLS: checked :ballot_box_with_check:
+    <details>
+      <summary>View Screenshot</summary>
+      <img alt="Screenshot of new alias popup" src="https://i.imgur.com/rh3YhK4.png" />
+    </details>
+5. Click "Add Account".
+
+We'll now send your first email to Mailinator, a cloud inbox that's helpful for email testing. Then you can confirm for yourself whether it works.
```

Comment:
This will tell md it's part of item 5, give it the right indentation and not restart numbering.

```suggestion
    We'll now send your first email to Mailinator, a cloud inbox that's helpful for email testing. Then you can confirm for yourself whether it works.
```

Reply:
I didn't wish it to be in with the last bullet

Reply:
I don't see why this isn't a step in itself, but not blocking.

Reply:
I re-read, and still feel this is comparable to indenting the list. Going to mark this resolved, since it's as-intended, but happy to talk about it next time we're chatting